### DISCUSSION TITLE
OK so I need help

### DISCUSSION THREAD

**Discussion by jchisholm59** [pfelk/pfelk#464] [General]
So at the risk of sounding really stupid, having gone through all the installation steps for the script version including the final steps..

How do I actually view the data from my pfSense?
Do I need to create a custom dashboard, and if so how ?

Thanks
Signed

Stupid

**Comment by jchisholm59** [comment]
Sorry.. I missed the last section re importing the dashboards..

**Reply by a3ilson** [reply]
@jchisholm59 - did you get everything working?

**Reply by jchisholm59** [reply]
Yes, thank you very much, I did.
What a wonderful job this package is.
Cheers
